Package.swift

```diff
@@ -1,34 +1,51 @@
+// swift-tools-version:4.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
 //
 //  BlueSocket.swift
 //  BlueSocket
 //
-//  Copyright © 2016 IBM. All rights reserved.
+//  Copyright © 2017 IBM. All rights reserved.
 //
-//     Licensed under the Apache License, Version 2.0 (the "License");
-//     you may not use this file except in compliance with the License.
-//     You may obtain a copy of the License at
+// 	Licensed under the Apache License, Version 2.0 (the "License");
+// 	you may not use this file except in compliance with the License.
+// 	You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+// 	http://www.apache.org/licenses/LICENSE-2.0
 //
-//     Unless required by applicable law or agreed to in writing, software
-//     distributed under the License is distributed on an "AS IS" BASIS,
-//     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//     See the License for the specific language governing permissions and
-//     limitations under the License.
+// 	Unless required by applicable law or agreed to in writing, software
+// 	distributed under the License is distributed on an "AS IS" BASIS,
+// 	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 	See the License for the specific language governing permissions and
+// 	limitations under the License.
 //
 
 import PackageDescription
 
 #if os(Linux) || os(macOS) || os(iOS) || os(tvOS)
-    
 let package = Package(
     name: "Socket",
-    targets: [Target(name: "Socket")],
-    exclude: ["BlueSocket.xcodeproj", "BlueSocket.xcworkspace", "README.md", "Sources/Info.plist", "Sources/Socket.h"]
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
```

Comment:
Can we delete lines like this boilerplate please?

Reply:
I've pushed an update that removes the stock comments, updates the copyright header to be consistent with the style of other IBM-Swift repos, and also removed the redundant (I believe?) `excludes` from the Socket target.